packages/hoppscotch-app/components/collections/graphql/Request.vue

```diff
@@ -177,13 +178,16 @@ export default defineComponent({
         this.pick()
       } else {
         setGQLSession({
```

Comment:
This doesn't handle the scenario where the previous version (the one which has no auth) GQL request data is selected. We have to have a migration mechanism for those requests.